src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -34,12 +34,23 @@
     <ItemGroup>
       <AspireProjectMetadataSource Update="@(AspireProjectMetadataSource)">
         <Source>
-        <![CDATA[namespace Projects%3B
-
+        <![CDATA[//------------------------------------------------------------------------------
```

Comment:
In the dotnet/runtime source generators we just generate:

```C#
// <auto-generated/>
```

https://github.com/dotnet/runtime/blob/4cf19ee05d3efc5372b8f308240688bacce09208/src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs#L32
https://github.com/dotnet/runtime/blob/4cf19ee05d3efc5372b8f308240688bacce09208/src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs#L135

These files go into the `obj` folder, I'm not sure we need the whole comment here.